todoapp-common/src/main/java/ru/todoapp/model/dto/AddTaskRequestDTO.java

```diff
@@ -0,0 +1,28 @@
+package ru.todoapp.model.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * Add task request.
+ * Allows for user to save new planned tasks for themselves
+ */
+@Getter
+@NoArgsConstructor
+public class AddTaskRequestDTO extends RequestDTO {
+    /**
+     * task description/name aka whatever the user is planning to do
+     */
+    private String description;
+
+    /**
+     * datetime, when the task is planned to take place, in ISO 8601 format (yyyy-MM-ddThh:mm:ss+hh)
+     */
+    private String datetime;
```

Comment:
Why you use here a string instead of long?
I thought we both agree, that we will use the UNIX time without a timezone

Reply:
We have tried to parse it to Instant (the field looked like: Instant datetime) and we were getting long from emulator, but I thought it was not super comfortable and changed it back in EmolatorDTO to String. 

Then I realized that we will not be able to have similar intafaces between AddTaskDTO (Instant datetime) and FetchTasksRequestDTO (String beginDate/endDate) and decided to go back to String.